examples/enums/enums.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/io;
+
+// Enums are defined using `enum` modifier.
+enum Color {
+  RED,
+  GREEN,
+  BLUE
+}
+
+// The value of enum members can be overridden in the enum declaration.
+enum Language {
+    ENG = "English",
+    TL = "Tamil",
+    SI = "Sinhala"
+}
+
+// The value of enum members can be a mixture of overridden and non-overridden members.
```

Comment:
```suggestion
// The value of the enum members can be a mixture of overridden and non-overridden members.
```

Reply:
Shall we change it to `The value of an enum member?`, it makes more sense imo.

Reply:
@pubudu91 @KRVPerera